device/simulation/tt_simulation_device_generated.h

```diff
@@ -21,36 +21,33 @@ struct DeviceRequestResponseBuilder;
 enum DEVICE_COMMAND : int8_t {
```

Comment:
This file seems to be generated from flatbuffer file `tt_simulation_device.fbs `. Do we even want to push the generated file to remote? I would assume that if you can always generate it from fbs we would just want to push the fbs to the repo

Reply:
The fbs is also pushed in the same repo. The reason I have the generated file pushed is because it gets rid of another step needed to setup the simulator. 

The flatbuffer structure itself shouldn't be changing often, so I don't expect this file to get updated often by anyone but me.

Reply:
The build flow will generate the `flatc` executable and then it needs to be manually run on the `.fbs` file to generate the header file. 

Actually, this could probably be made into a custom cmake target so someone can target it right after. Will look further into this.

Reply:
I will do this in a separate PR as it solely involves the build flow.

https://github.com/tenstorrent/tt-umd/pull/225